c7n/resources/cloudfront.py

```diff
@@ -35,6 +35,7 @@ class resource_type(object):
         date = 'LastModifiedTime'
         dimension = "DistributionId"
         universal_taggable = True
+        filter_name = None
```

Comment:
hmmm.. i'm wondering if we need a differential notion besides enumeration filters, getting all fronts on lambda to pick up one seems odd, if we have the ids technically we could just pass directly into detail spec to pick up the resources.